### DISCUSSION TITLE
Invert filter not working in Svg control

### DISCUSSION THREAD

**Discussion by kokaKOLA-NEW** [wieslawsoltes/Svg.Skia#233] [Q&A] [answered]
I'm currently using the Svg control in my project to display SVG images, and I'm trying to apply the invert filter to the images using the following code:
```xaml 
<Svg Path="/Assets/icon.svg" Css="* { filter: invert(1); }"/>
```

However, the invert filter is not working as expected. The images are not being inverted, and I'm not seeing any errors or warnings in the output window.

I've tested the invert filter with other controls and it works fine, so I'm wondering if there's a known issue with the Svg control and the invert filter.

Could someone please take a look at this and let me know if there's a workaround or a solution to this problem?

Thanks in advance for your help!

**Comment by wieslawsoltes** [comment, marked as answer]
Don't support CSS filter property

**Reply by kokaKOLA-NEW** [reply]
Ok, thanks you
